src/routers/handlers/upload/upload.ts

```diff
@@ -0,0 +1,96 @@
+import { logger } from "../../../utils/logger";
+import { GenericHandler } from "../generic";
+import { env } from "../../../config";
+import { fileIdPlaceholder,
+         servicePathPrefix,
+         uploadedUrl 
+} from "../../../utils/constants/urls";
+import { Request, Response } from "express";
+import { ContextKeys } from "../../../utils/constants/context.keys";
+import { TransactionService } from "../../../services/external/transaction.service";
+import { AccountsFilingService } from "services/external/accounts.filing.service";
+
+export class UploadHandler extends GenericHandler {
+    constructor (private accountsFilingService: AccountsFilingService, private transactionService: TransactionService) {
+        super({
+            title: '',
+            backURL: `${servicePathPrefix}`
+        });
+    }
+
+    async execute (req: Request, _res: Response): Promise<string> {
+        logger.info(`GET Request to send fileId call back address`);
+
+        const companyNumber = this.getCompanyNumber(req);
+
+        const transactionId = await this.callTransactionApi(companyNumber);
+        const transactionIdKey = ContextKeys.TRANSACTION_ID
+        req.session?.setExtraData(transactionIdKey, transactionId);
```

Comment:
No need to set this variable here
```suggestion
        req.session?.setExtraData(ContextKeys.TRANSACTION_ID, transactionId);
```